api/controllers/game/join-and-start-new-game.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Join the two players into the new game
+ */
+module.exports = async function (req, res) {
+  try {
+    const { newGameId, pNum } = req.body;
```

Comment:
Instead of reading the game ID and pNum from the request body, I think it would make sense to query for the user by their player id (in their session) and read the game id and pNum them from that record in the database. That way we could use a request to this endpoint as a fallback in case you miss the socket event that has your game id and this one could give you what you need without clientside params for the data the client could be missing